ci/rocm/clang14_release.yml

```diff
@@ -14,6 +14,7 @@ rocm clang14 release deps:
 rocm clang14 release build:
   extends:
     - .build_common
+  timeout: 1 hour 30 minutes
```

Comment:
I think this might be too tight for ROCm: https://gitlab.com/cscs-ci/ci-testing/webhook-ci/mirrors/4700071344751697/7514005670787789/-/jobs/9186534693.

Reply:
Anyway fixed by #1291 😉